syncer/test/fixtures/webhook.pull_request.closed.json.ts

```diff
@@ -0,0 +1,678 @@
+import {PullRequestClosedEvent} from '@octokit/webhooks-types';
+
+export default {
```

Comment:
do you really need all this data for tests, or can you create an object with only the data you need?